coremltools/converters/mil/test_flexible_shape_inputs.py

```diff
@@ -84,6 +88,25 @@ def test_multiarray_input_rangedim(self, convert_to):
         assert spec.description.input[0].type.multiArrayType.shapeRange.sizeRanges[2].upperBound == 100
         _assert_torch_coreml_output_shapes(model, spec, traced_model, example_input)
 
+    @pytest.mark.parametrize("convert_to", ['neuralnetwork', 'mlprogram'])
+    def test_multiarray_input_rangedim_flatten(self, convert_to):
+        if convert_to == "mlprogram" and ct.utils._macos_version() < (12, 0):
+            return
+
+        example_input = torch.rand(1, 3, 50, 50) * 100
+        traced_model = torch.jit.trace(TestFlattenModule().eval(), example_input)
+
+        input_shape = ct.Shape(shape=(1, 3, ct.RangeDim(25, 100, default=45), ct.RangeDim(25, 100, default=45)))
+        model = ct.convert(traced_model,
+                           inputs=[ct.TensorType(shape=input_shape)],
+                           convert_to=convert_to)
+
+        spec = model.get_spec()
+        assert list(spec.description.input[0].type.multiArrayType.shape) == [1, 3, 45, 45]
```

Comment:
I don't really think we need these assert. We don't need to test whether flexible shapes are parsed correctly.